PubNub/Data/Managers/PNSubscriber.m

```diff
@@ -15,6 +15,10 @@
 #import <objc/runtime.h>
 #import "PNHelpers.h"
 
+#ifdef PGDROID
+#import <UIKit/UIKit.h>
```

Comment:
XCode complains the UIApplication notification is not defined. 
I checked in our code base and saw Orlando did something like this elsewhere.